lib/traject/config/contentsearch_config.rb

```diff
@@ -0,0 +1,67 @@
+$LOAD_PATH << File.expand_path('../..', __dir__)
```

Comment:
Was this file meant to be part of this PR?

Reply:
nope.